index.bs

```diff
@@ -390,11 +400,10 @@ spec: SHA2; urlPrefix: http://csrc.nist.gov/publications/fips/fips180-4/fips-180
 
   1.  Let |parsedMetadata| be the result of
       <a href="#parse-metadata">parsing |metadataList|</a>.
-  2.  If |parsedMetadata| is `no metadata`, return `true`.
-  3.  If |parsedMetadata| is the empty set, return `true`.
-  4.  Let |metadata| be the result of <a href="#get-the-strongest-metadata">
+  2.  If |parsedMetadata| is the empty set, return `true`.
+  3.  Let |metadata| be the result of <a href="#get-the-strongest-metadata">
       getting the strongest metadata from |parsedMetadata|</a>.
-  5.  For each |item| in |metadata|:
+  4.  For each |item| in |metadata|:
```

Comment:
According to the [map-iterate](https://infra.spec.whatwg.org/#map-iterate), would it be more appropriate to write a phrase in the form "For each key → value of map"?.